### DISCUSSION TITLE
System collections hide core fields

### DISCUSSION THREAD

**Discussion by AlessiaCorsini** [directus/directus#13493] [Archive]
To avoid confusion for the editors I would like to hide some not required fields eg: in directus_files >> description/location/storage.
Is it somehow possible? I do not see any options in the data model
Thank you!

**Comment by azrikahar** [comment]
We can't modify the existing fields via Data Model for system collections (other than adding fields), but you can remove their permission to `read` those fields, which in turns hides it for them both in the App and the API. 

Quick example clip:

https://user-images.githubusercontent.com/42867097/169950583-9b2d5a66-1650-4ca4-a49b-c9369d83c80f.mp4

**Reply by AlessiaCorsini** [reply]
Amazing, thank you so much!
